src/CMakeLists.txt

```diff
@@ -1,20 +1,10 @@
 # Compile flags as defined in global CMakeLists
+# Todo: replace with target properties
```

Comment:
is this TODO supposed to still be there?

Reply:
I'm not sure any more. We set pretty much the same flags for every target so maybe in this particular case global variables are the way to go, otherwise we would need to repeat a lot of code for setting the flags. I still don't like the fact that we touch the CMAKE_CXX_FLAGS family as it's apparently against good practices but I'm not yet sure what exactly is the "good" practice here and any hints on I've found on the Internet are too vague.